quoteIdentifier.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+function quoteIdentifier (value, type) {
+  const quote = type === 'mysql' ? '`' : '"'
+
+  const quoted = value.replace(new RegExp(quote, 'g'), `${quote}${quote}`)
+
+  return `${quote}${quoted}${quote}`
+}
```

Comment:
I'm not very familiar with the mysql module, but apparently you can use `??` instead of `?` and use identifiers are regular parameters: https://github.com/mysqljs/mysql#escaping-query-identifiers

Reply:
this would complicate the implementation and make it different between pg and mysql, therefore I'll stick to the same approach and using the double backticks, which is what the native mysql function does and is not tied to a specific driver.